src/eval/mod.rs

```diff
@@ -13,6 +13,7 @@ mod str;
 #[macro_use]
 mod value;
 mod args;
+pub mod datetime;
```

Comment:
any reason why this is public rather than using the typical re-export route? it's just one type, right?

Reply:
Just an oversight. :)